app/components/visualizer/main-viz/templates/DynamicPieChart.js

```diff
@@ -0,0 +1,134 @@
+// Configurator: a configuration abstraction which will serve
+// as an abstract configuration for common configuration patterns.
+// As of now, it is coupled to the DynamicPieChart, but eventually
+// this should be ripped out into into a more robust abstraction.
```

Comment:
Yea - it seems our VizComponents fall pretty well into the "is-a" inheritance paradigm; a base-class we extend from makes quite a lot of sense.